src/System.Management.Automation/engine/DefaultCommandRuntime.cs

```diff
@@ -46,6 +47,15 @@ public DefaultCommandRuntime(List<object> outputList)
         /// <param name="errorRecord">Error record instance to process.</param>
         public void WriteError(ErrorRecord errorRecord)
         {
+            // Since this ErrorRecord was passed to WriteError,
+            // that means it's a non-terminating ErrorRecord.
+            // For RemotingErrorRecord's IsTerminating is already set
+            // so we shouldn't overwrite it.
```

Comment:
I think this comment may be partially redundant, it's just describing what the code itself says. I don't think it's really needed. The bit about remoting is probably good to leave though.

Reply:
Where set? In Serialization? What about previous versions and breaking remoting?

Reply:
> where set?

It's set in this PR. Look at `Job.cs`

I'll get back to you on the Remoting part.